CharlieBackend.Business/Services/AccountService.cs

```diff
@@ -72,7 +72,14 @@ public async Task<Result<AccountDto>> GetAccountCredentialsAsync(AuthenticationD
             {
                 authenticationModel.Password = PasswordHelper.HashPassword(authenticationModel.Password, salt);
 
-                var foundAccount = _mapper.Map<AccountDto>(await _unitOfWork.AccountRepository.GetAccountCredentials(authenticationModel));
+                var cred = await _unitOfWork.AccountRepository.GetAccountCredentials(authenticationModel);
```

Comment:
please rename cred to accountCredentials for better understanding(code conventions)